src/typings/tacoLiveReload.d.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ *******************************************************
+ *                                                     *
+ *   Copyright (C) Microsoft. All rights reserved.     *
+ *                                                     *
+ *******************************************************
+ */
+
+/// <reference path="../typings/Q.d.ts" />
+
+declare module TacoLiveReload {
+
+    export function start(projectRoot: string, platforms: string[], options: LiveReloadOptions): Q.Promise<any>;
+    export function on(eventName: string, cb: (...args : any[]) => void): void;
+    export function isLiveReloadActive(): boolean;
+
+    export class Patcher {
+        constructor(projectRoot: string, platforms: string[]);
+        public removeCSP(): void;
+        public patch(serverURL: string): Q.Promise<any>;
+    }
+
+    export interface LiveReloadHandle {
+        reloadFile(file?: string): Q.Promise<any>;
```

Comment:
I am not clear on what file argument does? what is the behavior in a multipage application? what if page is not even loaded in the app yet?